delivery_carrier_label_seur/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2015 FactorLibre - Ismael Calvo <ismael.calvo@factorlibre.com>
+# © 2017 PESOL - Angel Moya <angel.moya@pesol.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Seur Deliveries WebService",
+    "summary": "Allows to generate shipping label for SEUR shipments.",
+    "version": "10.0.1.0.0",
+    "category": "Delivery",
+    "website": "http://factorlibre.com",
```

Comment:
Could you follow convention in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py?

And maybe add some missing fields (e.g.: development_status, maintainers)